### PR TITLE
Log changes, spelling corrections

### PR DESCRIPTION
**Pull request by MegalexRex**
changed errors to debug
spelling corrections
debug now shows parameters and results of queries

### CODE REVIEW

File: vent/sqldb/sqldb.go [hunk 1]
```diff
@@ -173,23 +139,14 @@ func (db *SQLDB) UpsertRows(tables map[string]Table) error {
 		db.log.Debug("msg", "Log event", "query", clean(query), "value", registers)
 		_, err = tx.Exec(query, registers, "")
 		if err != nil {
-			db.log.Error("msg", "Error inserting to _bosmamromt_log", "err", err)
+			db.log.Debug("msg", "Error inserting to _bosmamromt_log", "err", err)
 			return err
```
Comment (hiturria) [inline vent/sqldb/sqldb.go]: Spelling error in debug msg: should be "_bosmarmot.log"

File: vent/sqldb/sqldb.go [hunk 2]
```diff
@@ -287,17 +244,19 @@ func (db *SQLDB) GetRowData(tableName, condition string) (Table, error) {
 
 		err = rows.Scan(pointers...)
 		if err != nil {
-			db.log.Error("msg", "Error scanning data", "err", err)
+			db.log.Debug("msg", "Error scanning data", "err", err)
 			return table, err
 		}
 
+		db.log.Debug("msg", "Resulset", "value", fmt.Sprintf("%+v", containers))
+
```
Comment (hiturria) [inline vent/sqldb/sqldb.go]: Spelling error in debug msg: should be "ResultSet"

File: vent/sqldb/sqldb.go [hunk 3]
```diff
@@ -562,7 +521,7 @@ func (db *SQLDB) getTableStructure(tableName string) (Table, error) {
 	db.log.Debug("msg", "Query table strucutre", "query", clean(query), "value", [2]string{db.schema, tableName})
 	rows, err := db.DB.Query(query, db.schema, table.TableName)
```
Comment (hiturria) [inline vent/sqldb/sqldb.go]: Spelling error in debug msg: should be "Query table structure"

File: vent/sqldb/sqldb.go (hunk 1 above)
Comment (MegalexRex) [reply vent/sqldb/sqldb.go]: Ready

File: vent/sqldb/sqldb.go (hunk 3 above)
Comment (MegalexRex) [reply vent/sqldb/sqldb.go]: Ready

**Pull request merged**